addon/components/au-dropdown.hbs

```diff
@@ -18,18 +19,29 @@
     </span>
   </AuButton>
   <div
-    class="au-c-dropdown__menu {{this.alignment}} {{if this.dropdownOpen 'is-visible'}}"
-    role="menu"
-    tabindex="-1"
-    {{focus-trap
-      isActive=this.dropdownOpen
-      shouldSelfFocus=true
-      focusTrapOptions=(hash
-        clickOutsideDeactivates=this.clickOutsideDeactivates
-      )
+    {{this.floatingUi
+      this.referenceElement
+      this.arrowElement
+      defaultPlacement=this.alignment
+      options=this.floatingUiOptions
     }}
-    {{on "click" this.closeDropdown}}
+    class="au-c-dropdown__floater {{if this.dropdownOpen 'is-visible'}}"
```

Comment:
If we go for the `{{#if this.dropdownOpen}}` wrapper the is-visible class isn't needed anymore.